mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -108,6 +108,7 @@ CLASS net/minecraft/server/MinecraftServer
 		ARG 2 safemode
 	METHOD method_29740 startServer (Ljava/util/function/Function;)Lnet/minecraft/server/MinecraftServer;
 		ARG 0 serverFactory
+	METHOD method_30002 getOverworld ()Lnet/minecraft/class_3218;
```

Comment:
main world or overworld? should only be overworld if the dimension id is `minecraft:overworld`. If this is `worlds[0]` then should be main world.

Reply:
Pretty sure it specifically refers to the overworld.

Reply:
This method uses the `RegistryKey` for the Overworld specifically, so `getOverworld` is a good name :+1: @liach